spec/features/users_index_spec.rb

```diff
@@ -0,0 +1,13 @@
+# spec/features/users_index_spec.rb
+
+RSpec.describe 'UsersIndex', type: :feature do
+  before do
+    @user1 = User.create!(name: 'User 1', email: 'user1@example.com')
+    visit users_path
+  end
+
+  it 'redirects to user show page on click' do
+    click_on @user1.name
+    expect(page).to have_current_path(user_path(@user1))
+  end
+end
```

Comment:
- [ ] I can see you have only one test here whereas the project milestone requires 4 tests from this file, kindly add the remaining tests.
![image](https://github.com/jlberbesi/blog-app/assets/86200356/940e970c-56e9-4529-946b-1be1f3060455)

Reply:
![imagen](https://github.com/jlberbesi/blog-app/assets/88105477/64d6a07d-d727-46ee-880c-d1c94d5fe845)